bazel/src/test/data/googleapis/google/example/library/v1/BUILD.bazel.updated.baseline

```diff
@@ -77,11 +70,8 @@ java_gapic_library(
     rest_numeric_enums = True,
     service_yaml = "library_example_v1.yaml",
     test_deps = [
-        ":library_java_grpc",
-        "//google/cloud/location:location_java_grpc",
-        "//google/iam/v1:iam_java_grpc",
     ],
-    transport = "lightning",
+    transport = "rest",
```

Comment:
nit: For robustness, I think it's worth retaining a case with an unknown transport. That said, I concede it's low priority given that we're not currently envisioning expanding the set of transports we'll support.

Reply:
OK, I'll file an issue. It seems to me we should either validate accepted values or successfully process arbitrary values as best we can.

Reply:
Thanks! Got the issue.